### PR TITLE
translation of date-picker

### PR DESCRIPTION
**Pull request by carmencitaqiu**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by carmencitaqiu** [comment]
translation of date-picker component

File: examples/docs/es/date-picker.md [hunk 1]
```diff
@@ -92,14 +92,15 @@
 </style>
 
 ## DatePicker
+Uso Date Picker para fecha entrada.
```
Comment (coderdiaz) [inline examples/docs/es/date-picker.md]: No sería mejor, `Utiliza Date Picker para introducir fechas`?

Comment (carmencitaqiu) [reply examples/docs/es/date-picker.md]: vale

Comment (Leopoldthecoder) [reply examples/docs/es/date-picker.md]: Hi @carmencitaqiu, could you update this PR to reflect this change?

**Comment by Leopoldthecoder** [comment]
@carmencitaqiu @coderdiaz Is there something wrong here? Is it because you are speaking different kinds of Spanish?

**Comment by carmencitaqiu** [comment]
@coderdiaz Which line?

**Comment by coderdiaz** [comment]
Where are you from @carmencitaqiu?

**Comment by carmencitaqiu** [comment]
I'm a chinese FE programer and I have learned spanish for two years.I have passed the exam of DELE level B2.I hope to participate this translation to improve my spanish and to learn FE/element at the same time.

File: examples/docs/es/date-picker.md [hunk 2]
```diff
@@ -92,14 +92,15 @@
 </style>
 
 ## DatePicker
+Utiliza Date Picker para introducir fecha.
 
-Use Date Picker for date input.
+###  Entrar fecha
 
-###  Enter Date
+Básica fecha escoja moderado por 'día'.
```
Comment (coderdiaz) [inline examples/docs/es/date-picker.md]: For example, this is confusing, but we can correct it later

Comment (carmencitaqiu) [reply examples/docs/es/date-picker.md]: Maybe Fecha escoja básica moderado por 'day'. better?

Comment (coderdiaz) [reply examples/docs/es/date-picker.md]: I think this is better: `Selector de fecha básico limitado por día`

Comment (carmencitaqiu) [reply examples/docs/es/date-picker.md]: OK,thanks,I will modify the datetime-picker as well.

File: examples/docs/en-US/dialog.md [hunk 3]
```diff
@@ -77,11 +77,13 @@
 
 Informs users while preserving the current page state.
 
-### Basic usage
+### Uso básico
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/dialog.md]: Please do not edit this file.

File: examples/docs/es/date-picker.md [hunk 4]
```diff
@@ -92,14 +92,15 @@
...
+Utiliza Date Picker para introducir fecha.
 
-Use Date Picker for date input.
+###  Entrar fecha
 
-###  Enter Date
+Selector de fecha básico limitado por día.
 
-Basic date picker measured by 'day'.
+:::manifestación Ésta medición es decidido por el atributo 'type'.Puedes permitir rápidas opcíones por crear un `picker-options` objecto con propiedad `shortcuts`.La fecha minusválida es establecido por
```
Comment (Leopoldthecoder) [inline examples/docs/es/date-picker.md]:
```
:::demo
```

```
:::tip
```
```
:::warning
```

These are symbols for vue-markdown-loader to parse, so please don't translate them.

File: examples/docs/es/dialog.md [hunk 5]
```diff
@@ -272,24 +276,25 @@ Dialog's content can be centered.
...
+| visible   | visibilidad del Diálogo, apoya el modificador .sync | boolean | — | false |
+| title     | título de Diálogo. También se puede pasar con una ranura nombrada (ver la tabla siguiente) | string    | — | — |
+| width     | anchura de Diálogo| string    | — | 50% |
+| fullscreen | si el diálogo ocupa pantalla completa | boolean     | — | false |
 | top      | value for `margin-top` of Dialog CSS | string    | — | 15vh |
-| modal     | whether a mask is displayed | boolean   | — | true |
-| modal-append-to-body     | whether to append modal to body element. If false, the modal will be appended to Dialog's parent element | boolean   | — | true |
+| modal     | si se muestra una máscara | boolean   | — | true |
+| modal-append-to-body | si adjuntar modal al elemento de cuerpo. Si es falso,el modal se agregará al elemento principal de Diálogo | boolean   | — | true |
 | append-to-body     | whether to append Dialog itself to body. A nested Dialog should have this attribute set to `true` | boolean   | — | false |
```
Comment (Leopoldthecoder) [inline examples/docs/es/dialog.md]: There's more to translate from this line.

Comment (carmencitaqiu) [reply examples/docs/es/dialog.md]: I see,today I will finish the translation of lines left

**Pull request merged**